instagram/src/App.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react';
+import './less/App.less';
+import dummyData from './dummy-data.js';
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
+
+
+class App extends Component {
+	constructor() {
+		super();
+		this.state = {
+			posts: [],
+			searchInput: '',
+		};
+  }
+  
+  componentDidMount(){
+    this.setState({posts: dummyData})
+  }
+
+
+	handleSearch = (event) => {
```

Comment:
I think you may need to make a separate change handler for your search. If you throw a console log for the event on line 22.5, you'll see a weird error message about event pooling. I'm trying to figure out what it means really.